package.json

```diff
@@ -117,7 +117,7 @@
       {
         "key": "Delete",
         "command": "extension.vim_delete",
-        "when": "editorTextFocus && vim.active && vim.mode == 'Normal' && !inDebugRepl"
+        "when": "editorTextFocus && vim.active && !inDebugRepl"
```

Comment:
I think it still passes through to VSCode since none of our actions capture `<Del>` in InsertMode.